.gitignore

```diff
@@ -1,2 +1,7 @@
 bin/configlet
 bin/configlet.exe
+*.swp
+*.swo
```

Comment:
Would you add the swap files to your global gitignore file?

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore